addons/website_sale/models/product_template.py

```diff
@@ -188,7 +188,6 @@ def _get_sales_prices(self, pricelist):
                 price_reduce, product_taxes, taxes, self.env.company)
```

Comment:
I think the `_fix_tax_included_price_company` call can also cause a difference between `base_price` and `price_reduce` (and should be moved below as well).